backend/src/lib/local-storage/index.js

```diff
@@ -0,0 +1,30 @@
+import { LocalStorage } from 'node-localstorage'
+
+const storage = new LocalStorage('./local-cache')
+
+const Settings = {
+  getItem: (key) => {
+    const value = storage.getItem(key)
+    return value ? JSON.parse(value) : null
+  },
+  setItem: (key, value) => {
+    if (value) storage.setItem(key, JSON.stringify(value))
+  },
+  addToUniqueArray: (key, value, limit) => {
+    let ary = Settings.getItem(key) || []
+    if (ary.constructor !== Array) {
+      throw new Error(`addToUniqueArray: ${key} is currently NOT an Array`)
+    }
+    ary.push(value)
+
+    if (limit) {
+      ary = ary.reverse().slice(0, Number(limit)).reverse()
```

Comment:
(minor) I think `ary.splice(0, ary.length - Number(limit))` does the same thing in a single function call.